src/components/organisms/BannerAdmin/BannerAdmin.scss

```diff
@@ -0,0 +1,89 @@
+@import "scss/constants.scss";
+
+.BannerAdmin {
+  &__input-text {
+    text-align: start;
+    padding: $spacing--sm $spacing--lg;
+    font-size: $font-size--md;
+
+    &--large {
+      height: $announcement-banner-admin-height;
+    }
+  }
+
+  &__input-container {
+    margin-bottom: $spacing--lg;
+  }
+
+  &__button-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: $spacing--xl;
+  }
+
+  &__button {
+    width: $announce-button-width;
+  }
+
+  &__action-container {
+    margin-left: 50px;
+  }
+
+  &__checkbox {
+    justify-content: start;
+    margin-bottom: $spacing--lg;
+
+    &--action {
+      margin-bottom: $spacing--xs;
+    }
+  }
+
+  &__checkbox__label {
+    font-size: $font-size--lg;
+  }
+  &__checkbox__label__disabled {
+    color: --white;
+    opacity: 20%;
+  }
+
+  & .BannerAdmin__button {
+    background-color: $primary--live;
+
+    &--close {
+      background-color: $red;
+    }
+  }
+}
+
+.BannerAdminModal {
```

Comment:
we should revise if we need that custom class name. I guess the confirmation modals should look more or less the same.

Reply:
probably, but this announcement banner differs from others according to the design

Reply:
do we have this class name now?